### ISSUE TITLE
Initialize Database

### ISSUE THREAD

**Issue by qy21gafy** [amosproj/amos2022ws05-shared-desk-mgmt#29]
## User story
1. As a user
2. I want / need a database that holds my user data, booked desks, company data etc.
3. So that I can efficiently book desks

## Acceptance criteria
* Database initialization should follow the specified design
* Database should contain dummy data to allow easy testing of future features
* SQL Database should be used, industry partner suggested MS-SQL-Server Express or PostgreSQL
* Dev environment setup in docker
* Research production databases 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
As the database design fits our idea, some suggestions for dummy data: Add an admin role, normal user and an intial admin role (user that registers company with service and has the most permissions) to the roles for testing; Add a dummy company with maybe one normal registered user, one admin and one initial admin; Maybe add one building, one room and one desk or so registered with that company to the database. These are just some suggestions, feel free to add more or less.

xilef45: Pull-Request: init DB + add Dummy Data https://github.com/amosproj/amos2022ws05-shared-desk-mgmt/pull/5
